backend/TreeOfAKind.Infrastructure/Domain/UserProfiles/UserProfileTypeConfiguration.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using TreeOfAKind.Application.Configuration;
+using TreeOfAKind.Domain.UserProfiles;
+
+namespace TreeOfAKind.Infrastructure.Domain.UserProfiles
+{
+    public class UserProfileTypeConfiguration : IEntityTypeConfiguration<UserProfile>
+    {
+        private const int MaxFirebaseAuthIdLength = 128;
+        public void Configure(EntityTypeBuilder<UserProfile> builder)
+        {
+            builder.ToTable("UserProfiles");
+
+            builder.HasKey(u => u.Id);
+
+            builder.HasIndex(u => u.AuthUserId);
+
+            builder.Property(u => u.AuthUserId)
```

Comment:
Why isn't it done in model using attributes?

Reply:
-1